### PR TITLE
version bump

### PR DESCRIPTION
**Pull request by sachaarbonel**
Hi @osyoyu this is a version bump of tantivy crate to 0.10.1

### CODE REVIEW

**Comment by osyoyu** [comment]
thanks! I'll take a look at this tomorrow 😉

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tantivy-tokenizer-tiny-segmenter"
-version = "0.1.0"
+version = "0.1.1"
```
Comment (osyoyu) [inline Cargo.toml]: Can you remove the version bump of tantivy-tokenizer-tiny-segmenter here? I just prefer to keep these independent of PRs

**Review by osyoyu — COMMENTED** [review]
Thanks for your contribution and sorry for being late 😄 Can you check out the points commented on? @Sach97

**Comment by sachaarbonel** [comment]
No problem @osyoyu and sorry for the delay I've just seen the notification. I've done the requested changes.

**Review by osyoyu — APPROVED** [review]
LGTM 😎

**Comment by osyoyu** [comment]
@Sach97 Just cut a new release https://crates.io/crates/tantivy-tokenizer-tiny-segmenter/0.3.0

**Comment by sachaarbonel** [comment]
Nice! Thank's @osyoyu

**Comment by sachaarbonel** [comment]
@osyoyu maybe we could setup a dependabot for future (non breaking) versions upgrade of tantivy ?

**Pull request merged**